src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/PlanControllerTest.kt

```diff
@@ -80,8 +80,8 @@ class PlanControllerTest : IntegrationTestBase() {
         .header("Content-Type", "application/json")
         .headers(setAuthorisation(user = "Tom C", roles = listOf("ROLE_RISK_INTEGRATIONS_RO")))
         .exchange()
-        .expectStatus().isOk
-        .expectBodyList<GoalEntity>().hasSize(0)
+        .expectStatus().isNotFound
+        .expectBodyList<ErrorResponse>()
```

Comment:
Test description above does not match what happens here.

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/55/commits/040d4f52592f085634c6b6282c1411274d3cf5d3